spec/controllers/container_group_controller_spec.rb

```diff
@@ -36,4 +36,25 @@
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
   end
+
+  describe "#show" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @container_group = FactoryGirl.create(:container_group)
+      @user = FactoryGirl.create(:user)
+      login_as @user
+    end
+
+    subject { get :show, :id => @container_group.id }
+
+    context "respond_with" do
+      it { is_expected.to have_http_status 200 }
+    end
+
+    context "render listnav partial" do
+      render_views
```

Comment:
So you are running the same `get` twice. Once with render views and once w/o render views to do 2 trivial tests.  And you want to do that will all the listnav tests. Why run it twice? What's the benefit?

Do I need to measure how much time it would take to run the extra test w/o render_view for all the listnavs to prove the point?

Readability: It's not difficult to read, but it's less obvious when it's 2 tests compared to having the 2 together. At least for me. Sure enough you can find a differend reviewer with a different idea.